nearuplib/nodelib.py

```diff
@@ -643,7 +636,7 @@ def stop_native():
             for line in f.readlines():
                 pid, proc_name, _ = map(str.strip, line.strip(' \n').split("|"))
                 pid = int(pid)
-                if proc_name in proc_name_from_pid(pid):
+                if proc_name == psutil.Process(proc.pid).name():
```

Comment:
Is the result from `psutil.Process(proc.pid).name()` the same as the result of `proc_name_from_pid(pid)`?
If it isn't I see a temporary issue for users who try to stop their node and after the update they will be unable to do so since names will not match, probably requiring manual intervention to stop their nodes.

A possible solution is temporary (at least two betanet deploys / two weeks) using the following approach:

```suggestion
                if proc_name in [psutil.Process(proc.pid).name(), proc_name_from_pid(pid)]:
```

Another option is try this by ourself and document this error (when transitioning from one nearup version to another) and make an issue with answer to link to validators and users who face this issue.

Reply:
Install with virtualenv & pip is a good approach, just make sure mention install pip3 and virtualenv in README.
Alternative approach is include all packages as git submodule or tarballs with nearup

Reply:
It should be the same, string in both cases.

Reply:
As well, yes if somebody wants they will install virutalenv as its standard practice to do so. will add it to the readme